src/qt/forms/sendcoinsentry.ui

```diff
@@ -144,6 +144,9 @@
       <property name="toolTip">
        <string>Enter a label for this address to add it to the list of used addresses</string>
       </property>
+      <property name="placeholderText">
+       <string>Enter a label for this address to add it to your address book</string>
```

Comment:
It's kind of strange to have the same thing worded in two different ways here. We might want to change this message to "Enter a label for this address to add it to the list of used addresses" as well (I think that's preferred to "address book"?). If people agree I'll do this in a new commit.

Reply:
> If people agree I'll do this in a new commit.

Yes, please.

Reply:
thx, done